### PR TITLE
Beta-1 and RC-1 release notes, with update to the roadmap README.MD

### PR DESCRIPTION
**Pull request by iamsamirzon**

### CODE REVIEW

File: RELEASENOTES/v2.0.0.rc-1.MD [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# Notary v2 Release Candidate 1 
+
+## Overview
+Notary v2 working group is pleased to announce its first release candidate, RC-1. Refer release definitions [here](https://github.com/notaryproject/notation/blob/main/RELEASE_MANAGEMENT.md). This release includes updates to the [notaryproject/notaryproject](https://github.com/notaryproject/notaryproject) repo to provide the community with updated specifications and an updated implementation of Notation client for signing and verifying artifacts.
+
```
Comment (yizha1) [inline RELEASENOTES/v2.0.0.rc-1.MD:5]: The tag will be `https://github.com/notaryproject/notation-core-go/releases/tag/v1.0.0-rc.1`

File: RELEASENOTES/v2.0.0.rc-1.MD [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+- [Notation Library 1](https://github.com/notaryproject/notation-core-go) ***[(notation-core-go-v1.0.0-rc.1)](https://github.com/notaryproject/notation-core-go/releases/tag/v0.1.0-rc.1)***   (Updated in this release)
```
Comment (yizha1) [inline RELEASENOTES/v2.0.0.rc-1.MD:6]: The tag will be https://github.com/notaryproject/notation-core-go/releases/tag/v1.0.0-rc.1

File: RELEASENOTES/v2.0.0.rc-1.MD [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+- [Notation Library 2](https://github.com/notaryproject/notation-go) ***[(notation-go-v1.0.0-rc.1)](https://github.com/notaryproject/notation-go/releases/tag/v1.0.0-rc.1)***   (Updated in this release)
+- [Notation CLI](https://github.com/notaryproject/notation)   ***([notation-v1.0.0-rc.1)](https://github.com/notaryproject/notation/releases/tag/v1.9.0-rc.1)***   (Updated in this release)
```
Comment (yizha1) [inline RELEASENOTES/v2.0.0.rc-1.MD:8]: The tag is `https://github.com/notaryproject/notation/releases/tag/v1.0.0-rc.1`

File: RELEASENOTES/v2.0.0.rc-1.MD [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+- [Notation Library 1](https://github.com/notaryproject/notation-core-go) ***[(notation-core-go-v1.0.0-rc.1)](https://github.com/notaryproject/notation-core-go/releases/tag/v0.1.0-rc.1)***   (Updated in this release)
+- [Notation Library 2](https://github.com/notaryproject/notation-go) ***[(notation-go-v1.0.0-rc.1)](https://github.com/notaryproject/notation-go/releases/tag/v1.0.0-rc.1)***   (Updated in this release)
+- [Notation CLI](https://github.com/notaryproject/notation)   ***([notation-v1.0.0-rc.1)](https://github.com/notaryproject/notation/releases/tag/v1.9.0-rc.1)***   (Updated in this release) 
+- [Notary v2 Specs](https://github.com/notaryproject/notaryproject) ***[(notaryproject-1.0.0-rc.1)](https://github.com/notaryproject/notaryproject/releases/tag/v1.0.0-draft.3)*** (Updated in this release)
```
Comment (yizha1) [inline RELEASENOTES/v2.0.0.rc-1.MD:9]: We can also consider to cut rc.1 release for specs, so the tag is
`https://github.com/notaryproject/notaryproject/releases/tag/v1.0.0-rc.1`

File: RELEASENOTES/v2.0.0.rc-1.MD [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+    - Configure Trust Policy as a JSON document, with support for registry scope and signature verfication levels to customize the behaviour during verification.
+    - Manage signatures in registries compliant to the OCI [1.1.0-rc2 image spec](https://github.com/opencontainers/image-spec/releases/tag/v1.1.0-rc2) and OCI [1.1.0-rc1 distribution spec] ([https://github.com/oras-project/oras-go/releases/tag/v2.0.0-rc.2](https://github.com/opencontainers/distribution-spec/releases/tag/v1.1.0-rc1)
+  - Updated libraries for Notation
+    - Both Notation-go and Notaiton-core-go libraries have been updated.
+    - Users can directly integerate with notation-go for signing and verification APIs. These APIs are made configurable to support verification of pre-fetched signatures one at a time.  
+*This is the first supported release from the Notary v2 community. Refer release definitions [here](https://github.com/notaryproject/notation/blob/main/RELEASE_MANAGEMENT.md). We suggest using it for testing and providing feedback to the Notary v2 working group only*
```
Comment (yizha1) [inline RELEASENOTES/v2.0.0.rc-1.MD:26]: wrong indent of this sentence

File: RELEASENOTES/v2.0.0.rc-1.MD [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+    - Verify signatures using Trust stores and Trust policy configuration of Notation client
+    - Sign images and Verify signatures with locally stored test keys/certificates for demonstration use only
+    - Setup Trust stores with the new directory-based structure using the notation client "certificate" commands to add/delete/list/show
+    - Configure Trust Policy as a JSON document, with support for registry scope and signature verfication levels to customize the behaviour during verification.
+    - Manage signatures in registries compliant to the OCI [1.1.0-rc2 image spec](https://github.com/opencontainers/image-spec/releases/tag/v1.1.0-rc2) and OCI [1.1.0-rc1 distribution spec] ([https://github.com/oras-project/oras-go/releases/tag/v2.0.0-rc.2](https://github.com/opencontainers/distribution-spec/releases/tag/v1.1.0-rc1)
+  - Updated libraries for Notation
+    - Both Notation-go and Notaiton-core-go libraries have been updated.
+    - Users can directly integerate with notation-go for signing and verification APIs. These APIs are made configurable to support verification of pre-fetched signatures one at a time.
```
Comment (yizha1) [inline RELEASENOTES/v2.0.0.rc-1.MD:25]: nit: one -> once

File: RELEASENOTES/v2.0.0.rc-1.MD [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+  - Updated libraries for Notation
+    - Both Notation-go and Notaiton-core-go libraries have been updated.
+    - Users can directly integerate with notation-go for signing and verification APIs. These APIs are made configurable to support verification of pre-fetched signatures one at a time.  
+*This is the first supported release from the Notary v2 community. Refer release definitions [here](https://github.com/notaryproject/notation/blob/main/RELEASE_MANAGEMENT.md). We suggest using it for testing and providing feedback to the Notary v2 working group only*
+
+## List of feature/capabilities added with this release
+For detailed notes on each of the individual repositories (NotaryProject, Notation (for reference CLI implementation) and Notation libraries) users can refer to the individual release notes. A summary view is provided in this document.
+
+### Notation CLI Client
+Notation is a standalone CLI client based on Notary v2 specs. It supports signing, verifying, and storing signatures in registries supporitng signatures as references to images.
```
Comment (yizha1) [inline RELEASENOTES/v2.0.0.rc-1.MD:32]: nit: supporitng -> supporting

File: RELEASENOTES/v2.0.0.rc-1.MD [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+## List of fixed issues
+Refer individual release notes.
+
+## Roadmap Guidance
+*This is not an exhaustive list. Users should refer to the Notary v2 roadmap at https://github.com/notaryproject/roadmap#readme for seeing the plan for subsequent releases and refer to individual subprojects (repositories) releases notes for details*
+
+1. Alternate signature storage schemes which can use OCI Distribution 1.0 based registries ("fallback support") is planned for the next release candidate
+2. Revocation support via Notation client will come in a future release. Spec update for revocation is planned for the next release candidate
+
```
Comment (yizha1) [inline RELEASENOTES/v2.0.0.rc-1.MD:61]: It is worth mentioning the CLI spec for managing trust policy is planned in next release, which improves the UX for managing trust policy easily and efficiently.

File: RELEASENOTES/v2.0.0.rc-1.MD [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+
+### Backwards Compatibility
+This version of Notation client is not compatible with any of the prior releases.
```
Comment (yizha1) [inline RELEASENOTES/v2.0.0.rc-1.MD:64]: This version works only with registries compatible with OCI IMAGE 1.1 spec

File: RELEASENOTES/v2.0.0.rc-1.MD (hunk 2 above)
Comment (iamsamirzon) [reply RELEASENOTES/v2.0.0.rc-1.MD:6]: Fixed

File: RELEASENOTES/v2.0.0.rc-1.MD (hunk 3 above)
Comment (iamsamirzon) [reply RELEASENOTES/v2.0.0.rc-1.MD:8]: Fixed

File: RELEASENOTES/v2.0.0.rc-1.MD (hunk 4 above)
Comment (iamsamirzon) [reply RELEASENOTES/v2.0.0.rc-1.MD:9]: Fixed.

File: RELEASENOTES/v2.0.0.rc-1.MD (hunk 6 above)
Comment (iamsamirzon) [reply RELEASENOTES/v2.0.0.rc-1.MD:25]: I think it is a style thing. Lets stick with the use of one.

File: RELEASENOTES/v2.0.0.rc-1.MD (hunk 5 above)
Comment (iamsamirzon) [reply RELEASENOTES/v2.0.0.rc-1.MD:26]: Fixed

File: RELEASENOTES/v2.0.0.rc-1.MD (hunk 7 above)
Comment (iamsamirzon) [reply RELEASENOTES/v2.0.0.rc-1.MD:32]: fixed

File: RELEASENOTES/v2.0.0.rc-1.MD (hunk 8 above)
Comment (iamsamirzon) [reply RELEASENOTES/v2.0.0.rc-1.MD:61]: ack. Done

File: RELEASENOTES/v2.0.0.rc-1.MD (hunk 9 above)
Comment (iamsamirzon) [reply RELEASENOTES/v2.0.0.rc-1.MD]: fixed

**Review by vaninrao10 — APPROVED** [review]
LGTM

**Review by yizha1 — APPROVED** [review]
LGTM

**Review by FeynmanZhou — APPROVED** [review]
LGTM

**Review by NiazFK — APPROVED** [review]
LGTM

**Review by SteveLasker — APPROVED** [review]
LGTM

**Pull request merged**